tests/test_ibmq.py

```diff
@@ -166,6 +171,70 @@ def circuit(theta, phi):
     assert np.allclose(res, expected, **tol)
 
 
+@pytest.mark.parametrize("shots", [1000])
+def test_simple_circuit_with_batch_params(token, tol, shots, mocker):
+    """Test that executing a simple circuit with batched parameters is
+    submitted to IBMQ once."""
+    IBMQ.enable_account(token)
+    dev = IBMQDevice(wires=2, backend="ibmq_qasm_simulator", shots=shots)
```

Comment:
Should we add a hardware backend?

Reply:
I don't think that's necessary. Here we test that the `IBMQDevice` works well, the rest should be up to IBMQ (i.e., if it works on a remote simulator, it should work on hardware too, unless we have different logic internally).